pootle/core/initdb.py

```diff
@@ -295,3 +306,56 @@ def create_default_languages(self):
                 self._create_object(Language, **criteria)
             except:
                 pass
+
+    def create_default_languages_cldr(self):
+        """
+        Generate Language objects from CLDR database
+
+        You can optionally set up a list of 'fallback aliases' (mappings from language definitions existing in CLDR
+        to locale names which you need in your Pootle installation)
+        """
+        from pootle_language.models import Language
+        for attrs in self.get_cldr_languages():
+            code = attrs.pop('code')
+            self._create_object(Language, code=code, defaults=attrs)
+
+
+    def update_languages_to_cldr(self):
```

Comment:
@unho Correct. Not sure it we should keep it or remove, actually. I myself used this function to convert my current database to CLDR-compliant (called manually from `pootle shell`), and probably someone else can find it useful as well... 

I can expose it as an option to `initdb` management command, or to ensure `initdb --cldr` always calls this function too.